<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Widget</class>
 <widget class="QWidget" name="Widget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>485</width>
    <height>405</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Widget</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout_4">
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <widget class="QGroupBox" name="groupBox">
       <property name="title">
        <string>Choose your fav OS(Exclusive)</string>
       </property>
       <layout class="QVBoxLayout" name="verticalLayout">
        <item>
         <widget class="QCheckBox" name="windowsCheckbox">
          <property name="text">
           <string>Windows</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QCheckBox" name="linuxCheckbox">
          <property name="text">
           <string>Linux</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QCheckBox" name="macCheckbox">
          <property name="text">
           <string>Mac</string>
          </property>
         </widget>
        </item>
       </layout>
      </widget>
     </item>
     <item>
      <widget class="QGroupBox" name="groupBox_2">
       <property name="title">
        <string>Choose Drink(Non Exclusive)</string>
       </property>
       <layout class="QVBoxLayout" name="verticalLayout_2">
        <item>
         <widget class="QCheckBox" name="coffeeCheckbox">
          <property name="text">
           <string>Coffee</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QCheckBox" name="teaCheckbox">
          <property name="text">
           <string>Tea</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QCheckBox" name="gingerteaCheckbox">
          <property name="text">
           <string>Ginger Tea</string>
          </property>
         </widget>
        </item>
       </layout>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <widget class="QGroupBox" name="groupBox_3">
     <property name="title">
      <string>Choose Gender</string>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout_3">
      <item>
       <widget class="QRadioButton" name="maleRadiobutton">
        <property name="text">
         <string>Male</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QRadioButton" name="femaleRadiobutton">
        <property name="text">
         <string>Female</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QRadioButton" name="otherRadiobutton">
        <property name="text">
         <string>Other</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="QPushButton" name="grabData">
     <property name="text">
      <string>Submit Info</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QPushButton" name="setStateButton">
     <property name="text">
      <string>Set State</string>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
